docs/shared/pods-setup-codegen-panel.mdx

```diff
@@ -0,0 +1,66 @@
+<ExpansionPanel title="Cocoapods">
+
+Apollo iOS compiles the Codegen CLI during `pod install`. This ensures you always have the version of the CLI for the version of Apollo iOS you are using.
+
+The Codegen CLI is located in your project at `Pods/Apollo/apollo-ios-cli`
+
+<ExpansionPanelList>
+<ExpansionPanelListItem>
+
+#### Initialize the codegen configuration
+
+The Codegen CLI uses a JSON file to configure the code generation engine. Initializing the Codegen CLI creates this file with default values. From the directory of your `Podfile`, run:
+
+```bash
+./Pods/Apollo/apollo-ios-cli init --schema-name ${MySchemaName}
+```
+
+`${MySchemaName}` should be the name you would like to be used as the namespace for your generated schema files.
+
+This will create an `apollo-codegen-config.json` file with default options.
+
+</ExpansionPanelListItem>
+
+<ExpansionPanelListItem>
+
+#### Set codegen configuration options
+
+Open the `apollo-codegen-config.json` file to configure code generation for your project.
+
+The default configuration will:
+- Find all GraphQL schema files ending with the file extension `.graphqls` within your project directory.
+- Find all GraphQL operation and fragment definition files ending with the file extension `.graphql` within your project directory.
+- Generate your generated schema files in a directory with the `schema-name` provided.
```

Comment:
```suggestion
- Create your generated schema files in a directory using the `schema-name` provided.
```

Reply:
The word using here doesn't seem clearly accurate to me. What I'm expressing is that the name of the directory in your folder structure will be the provided `schema-name`. Using seems too vague of a term to express that.

Reply:
That makes sense to me! I'd still suggest switching out the first word of this bullet point, though:
```suggestion
- Create your generated schema files in a directory with the `schema-name` provided.
```